go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/aws/amazon-vpc-cni-k8s
 go 1.12
 
 require (
-	github.com/aws/aws-sdk-go v1.21.7
+	github.com/aws/aws-sdk-go v1.23.13
```

Comment:
just curious, but what does this new version bring us?

Reply:
Pretty minor bug fix, but since we use the retry functionality I noticed that they didn't add up. Saw this in the [release notes](https://github.com/aws/aws-sdk-go/releases/tag/v1.23.13) 

* `aws/client`: Updates logic for request retry delay calculation ([#2796](https://github.com/aws/aws-sdk-go/pull/2796))
    * Updates logic for calculating the delay after which a request can be retried. Retry delay now includes the Retry-After duration specified in a request. Fixes broken test for retry delays for throttled exceptions.

Reply:
ack. ok, cool.